Automattic/jetpack
# VideoPress: re organize client files

This PR moves the files from the `./js` folder into the `./client` one.
There is only one file into `./js` so it's a matter to update the path where the file is being required./

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: re organize client files

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with a Jetpack site.
* Enable VideoPress module
* Go to the wp-admin page of your testing site
* Open network tab
* Select the js files and/or filter by `token` word
<img width="553" alt="Screen Shot 2022-08-17 at 09 55 50" src="https://user-images.githubusercontent.com/77539/185138424-8a7594e2-8467-49b1-9e1b-4b98252acf12.png">
* Confirm you see the file properly loaded
* Check it by doing a double click on it.
* It should show the file content

Note: probably you'll need to clear the cache of your browser
<img width="404" alt="image" src="https://user-images.githubusercontent.com/77539/185137912-1eea3eae-7ed5-499f-8e6a-f0fead352310.png">




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202798948032536
